source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -13,8 +13,8 @@ Document Data Format: POJOs
 Overview
 --------
 
-This guide explains how you can store and retrieve MongoDB data using plain
-old Java objects (`POJOs <https://en.wikipedia.org/wiki/Plain_old_Java_object>`__).
+In this guide, you can learn how you can store and retrieve data in the
```

Comment:
```suggestion
In this guide, you can learn how to store and retrieve data in the
```
Suggestion: "you can learn how you can" I think should changed to "you can learn how to"